draft-yun-privacypass-arc.md

```diff
@@ -175,9 +175,41 @@ where `pkI_serialized` is the serialized version of `pkI` as described in [Secti
 
 # Token Challenge Requirements {#token-challenge-requirements}
 
-Clients receive challenges for tokens as described in {{AUTHSCHEME}}.
-The HTTP authentication challenge also SHOULD contain the following
-additional attribute:
+The ARC protocol uses a modified TokenChallenge structure from the one
+specified in {{AUTHSCHEME}}. In particular, the updated TokenChallenge
+structure is as follows:
+
+~~~
+struct {
+    uint16_t token_type = 0xC7D3; /* Type ARC(P-384) */
+    opaque issuer_name<1..2^16-1>;
+    opaque origin_info<0..2^16-1>;
+    opaque credential_context<0..32>;
+    opaque redemption_context<0..32>;
+} TokenChallenge;
```

Comment:
It would be great to reorder to have a structure that extends the Token Challenge Structure defined in [Section 2.1.1 of RFC9577](https://datatracker.ietf.org/doc/html/rfc9577#name-token-challenge-structure)
```suggestion
    opaque issuer_name<1..2^16-1>;
    opaque redemption_context<0..32>;
    opaque origin_info<0..2^16-1>;
    opaque credential_context<0..32>;
} TokenChallenge;
```

Reply:
I thought about extending it, but opted not to since this is new parsing code based on the token type anyway. What do you see as the value in extending it?

Reply:
Mostly to match what exists. I agree this would be parsed anyway, but given this is the TokenChallenge it takes inspiration from at the moment could have the same order, I feel it makes sense to do it.

Reply:
I don't feel strongly, so I'm happy to take this.